packages/docs/next.config.js

```diff
@@ -14,12 +14,20 @@ module.exports = {
     BD_VERSION: bdPkg.version,
   },
   webpack: (config) => {
-    config.module.rules.push({
-      test: /\.(ts|tsx)$/,
-      use: [require.resolve('jsx-to-string-loader')],
-    });
-
-    return config;
+    return {
+      ...config,
+      module: {
+        ...config.module,
+        rules: [
+          ...config.module.rules,
+          {
+            test: /\.(ts|tsx)$/,
+            use: [require.resolve('jsx-to-string-loader')],
+          },
+        ],
+        noParse: /@babel\/standalone/,
```

Comment:
IIRC this issue was getting fixed in the next version of prettier, after `v2.3.2`. So prettier `v2.4.0` should do the trick.

PR that fixed it: https://github.com/prettier/prettier/pull/11130

Reply:
The issue we are having is with `@babel/standalone`

![Screen 2021-09-13 at 16 52 06](https://user-images.githubusercontent.com/2752665/133161377-d7ab5949-7a8d-4110-ae7c-a1aedec32ad4.png)

Reply:
Ah, guess I didn't dig deep enough when I looked at that 🤦